test/edu/csus/ecs/pc2/core/report/ContestCompareModelTest.java

```diff
@@ -195,12 +195,16 @@ public void testReport() throws Exception {
             // only test if there is a pretest dir found
```

Comment:
Three lines above this line there is a definition of a path which contains a hard-coded value of `c:\\repos...`.  (I'm commenting on that line here because for some reason GitHub is not letting me attach a comment to that line above.)

I cannot think of any valid reason why there should be a Windows-specific path (starting with "c:") in the code.  This will especially break because this is in the `test` suite, which will ultimately be run on the GitHub Linux build machine.